Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+#!groovy
+
+properties([
+        buildDiscarder(logRotator(numToKeepStr: '10', artifactNumToKeepStr: '2')),
+])
+
+node('linux') {
```

Comment:
The current version now works on any host but doesn't actually do a lot -- just runs the tests ensuring the resources (primarily the plugins blacklist) make sense.